sklearn/datasets/_base.py

```diff
@@ -164,11 +164,20 @@ def load_files(container_path, description=None, categories=None,
     Returns
     -------
     data : Bunch
-        Dictionary-like object, the interesting attributes are: either
-        data, the raw text data to learn, or 'filenames', the files
-        holding it, 'target', the classification labels (integer index),
-        'target_names', the meaning of the labels, and 'DESCR', the full
-        description of the dataset.
+        Dictionary-like object, with attributes.
```

Comment:
```suggestion
        Dictionary-like object, with the following attributes.
```

Please also update the other docstrings

Reply:
Thank you for your review. I'll update them.